### DISCUSSION TITLE
Search among the saved tabs

### DISCUSSION THREAD

**Discussion by Alunar05** [cxw42/TabFern#319] [General]
Hello,

After some months of use, the amount of saved tabs and folders has grown exponentially (and I like it that way!). So far, there are 2 features that I believe would be incredibly useful:

1. Search among all the tabs and folders
2. A feature to (search and) delete duplicates 

Thanks!

**Comment by cxw42** [comment]
Thanks for using TF, and for the feedback!  Search open tabs is #75 .  I am going to open a new issue (**edit** #320) re. managing duplicates.

Chris
